### PR TITLE
Remove batchnorm, weight decay, and SOS from transducer conformer encoder

### PR DESCRIPTION
**Pull request by csukuangfj**

### CODE REVIEW

**Comment by csukuangfj** [comment]
Here are the results for this PR.

## This PR

|                           | test-clean | test-other | comment                                  |
|---------------------------|------------|------------|------------------------------------------|
| greedy search             | 2.85       | 7.30       | --epoch 29, --avg 13, --max-duration 100 |
| beam search (beam size 4) | 2.83       | 7.19       |                                          |


## Before this PR, i.e., the master

|                           | test-clean | test-other | comment                                  |
|---------------------------|------------|------------|------------------------------------------|
| greedy search             | 2.99       | 7.52       | --epoch 20, --avg 10, --max-duration 100 |
| beam search (beam size 4) | 2.92       | 7.37       |                                          |
